crates/weaver_resolver/src/body.rs

```diff
@@ -0,0 +1,88 @@
+
+// SPDX-License-Identifier: Apache-2.0
+
+//! Functions to resolve a semantic convention body.
+
+use weaver_resolved_schema::body::{Body, BodyField};
+use weaver_semconv::body::{BodyFieldSpec, BodySpec};
+
+use crate::attribute::AttributeCatalog;
+
+/// Resolve a body specification into a resolved body.
+pub fn resolve_body(
+    body: &BodySpec,
+    attr_catalog: &mut AttributeCatalog
+) -> Option<Body> {
+    match body {
+        BodySpec::Fields { fields } => {
+            let fields = fields.iter().map(|field| resolve_body_field(field, attr_catalog)).collect();
+            Some(Body {
+                fields: Some(fields),
+                value: None,
+            })
+        },
+        BodySpec::Value { value: _ } => {
+            // Add as a placeholder for now of where to resolve the value.
+            todo!("Implement value type for body.")
```

Comment:
It is preferable to emit an explicit Rust error indicating that the body value is not yet supported. You can specify the level of severity of the error using the `miette` annotations, which in this case, could be set as a warning.